packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -130,14 +134,37 @@ export const Timeline: React.FC<TimelineProps> = ({
   })
 
   return (
-    <TimelineProvider startDate={startDate} today={today} options={options}>
-      <article className="timeline">
-        {rootChildren}
-        <div className="timeline__inner">
-          <header className="timeline__header">{headChildren}</header>
-          {bodyChildren}
-        </div>
-      </article>
+    <TimelineProvider
+      startDate={startDate}
+      endDate={endDate}
+      today={today}
+      options={options}
+    >
+      <TimelineContext.Consumer>
+        {({ state: { months } }) => {
+          const offset = endDate
```

Comment:
If I change this line to `const offset = null` then all tests still pass. I see your note about adding a test but it doesn't feel right when coverage is not compromised because of a ternary operator.

Reply:
I'll add a test that checks for the dynamic styles.